joern-cli/frontends/pysrc2cpg/src/test/scala/io/joern/pysrc2cpg/dataflow/DataFlowTests.scala

```diff
@@ -221,6 +221,81 @@ class DataFlowTests extends PySrc2CpgFixture(withOssDataflow = true) {
     sink2.reachableBy(pSrc).size shouldBe 1
   }
 
+  "Import statement with method ref sample one" in {
+    val controller =
+      """
+        |from django.contrib import admin
+        |from django.urls import path
+        |from django.conf.urls import url
+        |from student import views
+        |
+        |urlpatterns = [
+        |    url(r'^allPage/$', views.all_page)
+        |]
+        |""".stripMargin
+    val views =
+      """
+        |def all_page(request):
+        |	print("All pages")
+        |""".stripMargin
+    val cpg = code("print('Hello, world!')")
+      .moreCode(controller, "controller/urls.py")
```

Comment:
```suggestion
      .moreCode(controller, Seq("controller", "urls.py").mkString(File.separator))
```
`File` comes from `java.io.File`

Reply:
done